client/src/components/SignUpAd/SignUpAd.css

```diff
@@ -28,21 +26,21 @@
   background-color: #df7362;
 }
 
-
-
+.SignUpAd h2,
+h3 {
```

Comment:
Can you utilize the css naming convention? Also this `h3` styling will clash with other `h3`s in the app

Reply:
If I understood the notes, I think corrections should be all set, let me know otherwise!

Reply:
The naming all looks good now! Thank you